lib/kv_behaviour.ex

```diff
@@ -1,10 +1,13 @@
 defmodule Kv.Behaviour do
   @moduledoc false
 
+  @typedoc "Key-value storage"
+  @type kv :: %{optional(any()) => any()}
```

Comment:
❤

Reply:
Eventually I found a way to fix the warning  :tada: